cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -465,8 +472,8 @@ fn emit_fcmp<C: LowerCtx<I = Inst>>(
         (inputs[0], inputs[1])
     };
     let lhs = put_input_in_reg(ctx, lhs_input);
-    let rhs = input_to_reg_mem(ctx, rhs_input);
-    ctx.emit(Inst::xmm_cmp_rm_r(op, rhs, lhs));
+    let rhs = put_input_in_reg(ctx, rhs_input);
```

Comment:
Maybe a comment saying something like "see comment above `let rhs = put_input_in_reg(...)` in `emit_cmp` for why we force RHS into a register" because this is a bit subtle?

Reply:
Done, thanks!